components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/PassiveSTSFederatedAuthenticatorConfig.java

```diff
@@ -27,6 +27,15 @@
      *
      */
     private static final long serialVersionUID = 2265391150645470497L;
+    private String idpEntityId;
```

Comment:
Cannot use instance variable here. All the specific details need to go as properties. Check OpenIDConnectFederatedAuthenticatorConfig.